docs/Technical-Documentation/jwt-key-rotation.md

```diff
@@ -0,0 +1,74 @@
+# Rotating JWT Keys
+
+## Context
+
+To maintain good security, we will periodically rotate the JWT keys used to control authentication and authorization to our application. This document outlines the process of how to do this.
+
+**Warning** Production sites will need to be taken down for maintenance when rotating keys, as the rotation will automatically invalidate all current sessions.
+
+### 1. Generate New Keys
+
+In your Mac terminal (or bash terminal in Windows), enter the following command:
+```bash=
+ssh-keygen -t rsa -b 4096 -m PEM -f jwtRS256.key
+```
+You will receive the following response. Do not add a passphrase when prompted.
+```
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in jwtRS256.key
+Your public key has been saved in jwtRS256.key.pub
+The key fingerprint is:
+# returns key fingerprint
+The key's randmart image is:
+# returns key's randmart image
+```
+Then enter
+```bash=
+openssl rsa -in jwtRS256.key -pubout -outform PEM -out jwtRS256.key.pub
+```
+
+You can now check the contents of your keys with these commands
+```bash=
+cat jwtRS256.key
+# returns private key
+cat jwtRS256.key.pub
+# returns public key
+```
+
+### 2. Base64 Encode Private Key
+
+We use Base64 Encoded Private Keys to make it easier to save to cloud environments and local `.env` files.
+
+```bash=
+openssl enc -base64 -w 0 -in jwtRS256.key -out jwtRS256.key.base64
```

Comment:
@jtwillis92 what does `-w 0` mean? this returned an error on my end and `openssl enc -help` didnt shed any light. i removed this from the script and doesnt return any errors, but will this cause any problems for us?

Reply:
thanks @jtwillis92 can you update the doc w/ this distinction? I'll proceed with rotating and finishing up the qasp review.